test/JDBC/expected/db_collation/test_like_index-vu-verify.out

```diff
@@ -0,0 +1,972 @@
+
+-- psql
+-- VARCHAR
+-- CS_AS
+ANALYZE master_dbo.test_like_index_vu_prepare_v_csas;
+GO
+
+-- tsql
+SET BABELFISH_SHOWPLAN_ALL ON
+GO
+
+SELECT * FROM test_like_index_vu_prepare_v_csas WHERE test_like_index_vu_prepare_v_csas_col LIKE 'cs1'
```

Comment:
have test case which will verify that index scan wont be picked up if you supply explicit collation that doesnt match with column collation

Reply:
Added in latest rev